examples/jf-control-app/commands/pairing/PairingCommand.cpp

```diff
@@ -40,9 +40,18 @@ using namespace ::chip::Controller;
 CHIP_ERROR PairingCommand::RunCommand()
```

Comment:
Please add an integration test to verify this. We cannot rely on manual testing for such procedures.

Reply:
@paulux15 will submit an integration test shortly.